cmd/src/config.go

```diff
@@ -50,7 +50,7 @@ Use "src config [command] -h" for more information about a command.
 }
 
 const configurationSubjectFragment = `
-fragment ConfigurationSubjectFields on ConfigurationSubject {
+fragment ConfigurationSubjectFields on SettingsSubject {
```

Comment:
To avoid others being confused by this in the future, let's make the naming consistent 🙂

```suggestion
fragment SettingsSubjectFields on SettingsSubject {
```

Reply:
I dug a bit deeper and the changes turned out to be pretty-far reaching - all the `Configuration` fields are listed as deprecated, and the shape of some things have changed, so I went ahead and made everything `Settings`: 9a39f3e59ec73c7c282d3cdbdf73a501c65e4203